.codex/hooks/test_hook_adapters.py

```
@@ -0,0 +1,1096 @@
+PHASE_GUARD = load("guard_phase.py")
+STOP_GUARD = load("guard_stop.py")
+
+GIT_ENV = {
+    "GIT_CONFIG_GLOBAL": os.devnull,
+    "GIT_CONFIG_SYSTEM": os.devnull,
+    "GIT_TERMINAL_PROMPT": "0",
+}
+
+
+def git(cwd: pathlib.Path, *args: str, check: bool = True) -> subprocess.CompletedProcess[str]:
+    return subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)